jquery.facetview.js

```diff
@@ -1581,6 +1582,11 @@ search box - the end user will not know they are happening.
                     $(options.searchbox_class).bindWithDelay('keyup',dosearch,options.freetext_submit_delay);
                 }
 
+                // Properly set size for each facet
+                $.each(options.facets, function(facet){
+                  options.facets[facet]["size"] = options.facet_size;
```

Comment:
That's what I was thinking. I just wanted to make sure. That makes this problem a bit more complicated. 

When you change the number of entries, it makes a new request for the facet and gets exactly that amount of results, that is 10 instead of the initial 100. This seems to show that the data is being changed when you change the length of the filter.

It seems to me that the safest way to fix this is to re-render the filter element without making the request to the server, and without making changes to the data. This is how it should have been done. 

This will require some new functions and some refactoring. I think it's probably worthwhile to do a git blame and a comparison with the original code to figure out what it is supposed to be doing.

We can chat tomorrow if you need clarification on this.